hack/install-deps.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
i think the name of this file (and the make target) is too vague for whats being done here. This is actually installing the operator sdk. "install deps" seems like it would be for the operator itself, which is unnecessary.

Reply:
why not call this like "install-sdk" instead?

Reply:
The script is meant to be a placeholder for all dependencies but I think I will remove for the time being since I don't think its fully flushed out.